requirements-test.txt

```diff
@@ -0,0 +1,5 @@
+pytest==3.9.2
```

Comment:
All this should be in `requirements.txt`, it should install all that is needed for testing. Package dependencies go directly into `setup.py`.